example/deployment/AWS-EC2-S3/terraform-scripts/InstanceRunningExample/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "project" {
+  default = "Palisade Example"
+}
+
+variable "owner" {
+  default = "Unknown"
+}
+
+variable "aws_region" {
+    default = "eu-west-1"
+}
+
+variable "aws_access_key_id" {
+  default = ""
+}
+
+variable "aws_secret_access_key" {
+  default = ""
+}
+
+variable "aws_session_token" {
+  default = ""
+}
+
+variable "aws_profile_name" {
+  default = ""
+}
+
+variable "bucket_name" {}
+
+variable "s3_endpoint" {
+  default = ""
```

Comment:
remove the default, we don't want this terraform script to run with an empty s3 endpoint